setupext.py

```diff
@@ -1097,21 +1100,20 @@ def do_custom_build(self):
                     tarball_url = url_fmt.format(
                         version=LOCAL_FREETYPE_VERSION, tarball=tarball)
 
-                    print("Downloading {0}".format(tarball_url))
+                    print("Downloading {}".format(tarball_url))
                     try:
                         urllib.request.urlretrieve(tarball_url, tarball_path)
-                    except IOError:  # URLError (a subclass) on Py3.
-                        print("Failed to download {0}".format(tarball_url))
+                    except IOError:
```

Comment:
Changed in version 3.3: URLError has been made a subclass of OSError instead of IOError.
https://docs.python.org/3/library/urllib.error.html

We should check for the more specific `urllib.error.URLError` anyway.

Reply:
IOError is an alias of OSError since 3.3 (see just after https://docs.python.org/3/library/exceptions.html#ZeroDivisionError).